src/containers/ColorContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import Button from '../components/Button';
+import { ColorView } from '../components/ColorView';
+import useRandomValues from '../hooks/useRandomValues';
+
+const ButtonRapper = {
+  marginBottom: '20px',
+};
+const ColorContainer = () => {
+  const [randomValues, addBlock, deleteLastBlock] = useRandomValues();
+  return (
+    <div>
+      <div style={ButtonRapper}>
+        <Button label="Add block" onClick={() => addBlock()} />
+        <Button
+          disabled={randomValues?.length === 0}
```

Comment:
null operator is redundant here, you can event just check this condition via !randomValues.length